services/pdf-ms/helpers/user.ts

```diff
@@ -0,0 +1,22 @@
+import to from 'await-to-js';
+
+import config from '../../../config';
+import * as dynamoDb from '../../../libs/dynamoDb';
+
+export const getUser = async (personalNumber: string) => {
+  const params = {
+    TableName: config.users.tableName,
+    Key: {
+      personalNumber,
+    },
+  };
+  const [error, userGetResponse] = await to<{ Item: Record<string, any> }>(
+    dynamoDb.call('get', params)
+  );
+  if (error) {
+    console.error(error);
+    return undefined;
```

Comment:
Varför inte return false?

Reply:
Because it felt better like better typing to have either a json object or undefined, rather than a json or false. Sort of amounts to the same thing though.